app/views/liquids/_open_graph.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="crayons-card p-5">
+  <% if page.image %>
+    <div class="s:max-w-50 mb-4">
+      <img src="<%= page.image %>" height="auto" alt="Avatar">
```

Comment:
1. Let's change the `alt` attribute value as it's not really an avatar - but more of a _Cover image_ so it's more "accessible" to assistive technologies.
2. I haven't tested it yet on local env but I suspect we may want to make image sizing more bullet proof (but that's only my suspicion for now)

Reply:
re: image sizing, I would appreciate more thoughts on this @ludwiczakpawel.

I know there is a `max-height` set for images within a Post, BUT that setting is generous, which may result in the fallback embed allowing rather LONG images. 🙃